autoversion_lib/__init__.py

```diff
@@ -161,23 +169,28 @@ def commit2timestamp(self, commit: str) -> int:
 
 
 class Autoversion:
-    def __init__(self, path: str, precache: bool = False):
+    def __init__(self, path: str, precache: bool = False) -> None:
         try:
             git = Git(path)
         except GitError as e:
-            raise AutoverError(f"failed to initialize gitwrapper: {e}") from e
```

Comment:
Imo we should keep the "from e", and just format the error manually if it hits panic. That way we preserve the full stack-trace if it ever gets raised somewhere it's not caught.